kube/base-components/kibana-deployment.yaml

```
@@ -0,0 +1,74 @@
+          ports:
+            - containerPort: 5601
+          volumeMounts:
+            - name: kibana-config
+              mountPath: /usr/share/kibana/config/kibana.yml
+              subPath: kibana.yml
+          env:
+            - name: ELASTICSEARCH_HOSTS
+              value: "http://elasticsearch.default.svc.cluster.local:9200"
+            - name: ES_USERNAME
+              valueFrom:
+               secretKeyRef:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 16 but found 15

(indentation)